IntBuilder.js

```diff
@@ -0,0 +1,66 @@
+import BuilderES6 from './BuilderES6.js'
+
+// ES6 Way
+export default class IntBuilder extends BuilderES6 {
+  constructor (operand) {
+    super(operand ?? 0)
```

Comment:
I would personally use the default parameter instead

Reply:
replaced with default parameter